### PR TITLE
automate OCP-22596

### PR DESCRIPTION
**Pull request by wzheng1**
Here is test result: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/81518/console

### CODE REVIEW

**Comment by wzheng1** [comment]
@openshift/devexp-qe Please review, thanks@

File: features/build/stibuild.feature [hunk 1]
```diff
@@ -124,3 +124,14 @@ Feature: stibuild.feature
...
     And the output should contain "error"
 
+  # @author wzheng@redhat.com
+  # @case_id OCP-22596
+  Scenario: Create app with template eap72-basic-s2i with jbosseap rhel7 image
+    Given I have a project
+    When I run the :new_app client command with:
+      | template | eap72-basic-s2i |
+    Then the step should succeed
+    Given the "eap-app-1" build was created
```
Comment (pruan-rht) [inline features/build/stibuild.feature:134]: @wzheng1 is the `created` status check absolutely needed?  I find that intermediate status check to be hard to catch and could be a source of un-reliability for the test.

Comment (wzheng1) [reply features/build/stibuild.feature:134]: If the needed imagestream exits under openshift imagestream, the build should be created soon.

**Pull request merged**